lib/ts/recipe/thirdpartyemailpassword/components/themes/signInAndUp/index.tsx

```diff
@@ -34,6 +34,7 @@ import { ThemeBase } from "../themeBase";
 
```

Comment:
not related to this file, but you need to modify the ts playground to use the isNewPrimaryUser, and also check that all components that can be embedded by the user should accept a userContext + that getRedirection function also has a userContext as one of the args.

Reply:
done